tests/fixture_valid.js

```diff
@@ -308,6 +311,35 @@ module.exports.checkFixture = function checkFixture(fixture, usedShortNames=[])
   return result;
 };
 
+function checkCoarseChannelExistence(result, fixture, fineChannels, switchingChannels, ch, mode, chNumber) {
+  const coarseChannelKey = fineChannels[ch];
+
+  // the mode must also contain the coarse channel
+  if (!mode.channels.includes(coarseChannelKey)) {
```

Comment:
What if the coarse channel is used only in part of a switching channel?

E.g. `[Dimmer, Dimmer Fine, Programs]` can become `[Speed, Fade Time, Programs]`

You could just get rid of this special handling and use the logic below.